documentation/docs/pages/infrastructure/Building.md

```diff
@@ -0,0 +1,44 @@
+# Build infrastructure
+
+Most users of ACCESS-ESM1.6 will not need to build the model. Released versions of the model are deployed by ACCESS-NRI to Gadi and available to users. If you require your own build of the model, typically because you need to modify one or several model components, you will find the required information on this page.
+
+ACCESS-NRI is using [Spack](https://spack.io/), a build from source package manager designed for use with high performance computing. Spack already contains support for compiling thousands of common software packages. 
+
+## Software structure
+
+ACCESS-ESM1.6 being a Earth System model, it is constituted of several model components, each residing in its own repository on GitHub. In addition, some model components need other utilities at compilation time that are stored in separate repositories. This results in a complex software structure involving a large number of repositories:
+
+- deployment repository: 
+    - [ACCESS-NRI/ACCESS-ESM1.6](https://github.com/ACCESS-NRI/ACCESS-ESM1.6)
+- model components:
+    - [ACCESS-NRI/cice5](https://github.com/ACCESS-NRI/cice5)
+    - [ACCESS-NRI/GFDL-generic-tracers](https://github.com/ACCESS-NRI/GFDL-generic-tracers)
+    - [ACCESS-NRI/MOM5](https://github.com/ACCESS-NRI/MOM5)
+    - [ACCESS-NRI/UM7](https://github.com/ACCESS-NRI/UM7)
+    - [CABLE-LSM/CABLE](https://github.com/CABLE-LSM/CABLE)
+- other software:
+    - [ACCESS-NRI/FMS](https://github.com/ACCESS-NRI/FMS)
+    - [ACCESS-NRI/mocsy](https://github.com/ACCESS-NRI/mocsy)
+    - [ACCESS-NRI/GCOM4](https://github.com/ACCESS-NRI/GCOM4)
+
+!!! note
+
+    The _UM7_ and _GCOM4_ repositories are private because these software are not open-source. They are covered by the UKMO's Momentum Partnership licence. To gain access to these repositories, please request an invitation via [this ACCESS-Hive forum topic](https://forum.access-hive.org.au/t/request-access-to-access-esm1-6-component-repositories/5709).
+
+## Building ACCESS-ESM1.6 using the deployment infrastructure
```

Comment:
@aidanheerdegen @harshula , I'd appreciate if one of you could have a read through this section before the end of January. I want to make sure I don't misconstrue anything.